packages/components-react/src/select/Select.css

```
@@ -2,13 +2,21 @@
 .lux-select {
   line-height: var(--lux-line-height-body-default);
 
-  & :focus {
-    border-width: var(--lux-border-width-m);
+  &:not([multiple]) {
+    background-position: center right var(--utrecht-select-padding-inline-end);
+    background-size: var(--lux-size-icon);
   }
 
-  & :hover {
-    border-width: var(--lux-border-width-m);
-    border-color: var(--lux-border-strong);
+  &:focus {
+    box-shadow: 0 0 0 var(--lux-border-width-m) var(--lux-color-border-strong);
```

Graag component tokens toepassen zodat de stijling vanuit de json gewijzigd kan worden, ik zie hier nu 2 common tokens staan. Dus dan zou het dit zijn:

```suggestion
    box-shadow: 0 0 0 var(--org-prefix.select.focus.border-width) var(--org-prefix.select.focus.border-color);
```

De `border-width` was eerder een todo-token, maar het lijkt erop dat deze recent is toegevoegd aan het Utrecht-component. De `border-color` leek al wel te bestaan. Gebruiken wij mogelijk nog een oudere versie van het component, waardoor deze tokens niet _automatisch_ werken?

---

kan in deze pr de todo gebruiken en als we het hebben geupdate alleen naar utrecht hoeven te renamen? dit is inderdaad nog de oudere versie.

---

Graag todo veranderen in lux, dan is het top!